sagemaker/launch/launch_train.py

```diff
@@ -128,13 +131,14 @@ def get_train_parser():
     # task specific
     training_args.add_argument("--graph-name", type=str, help="Graph name",
         required=True)
-    training_args.add_argument("--graph-data-s3", type=str,
-        help="S3 location of input training graph", required=True)
     training_args.add_argument("--task-type", type=str,
         help=f"Task type in {SUPPORTED_TASKS}", required=True)
     training_args.add_argument("--yaml-s3", type=str,
         help="S3 location of training yaml file. "
              "Do not store it with partitioned graph", required=True)
+    training_args.add_argument("--node-mappings-s3", type=str,
```

Comment:
In train_infer you use --node-id-mappings-s3, but here you use --node-mappings-s3.
BTW, I don't think we need to update launch_train.py here.
In SageMaker, save-embedding and save-prediction are always set to False.